search-sentiments-map/public/scripts/map-script.js

```diff
@@ -26,9 +26,20 @@ let mapStyle = [{
 let map;
 let infowindow;
 
+const WORLD_CENTER_COORDINATES = {lat: 29.246630, lng: 29.678410};
+const WORLD_ZOOM_LEVEL = 3;
+const WORLD_GEOJSON = 'countries.geojson';
+const US_CENTER_COORDINATES = {lat: 39.844724, lng: -92.019078};
+const US_ZOOM_LEVEL = 5;
+const US_GEOJSON = 'https://storage.googleapis.com/mapsdevsite/json/states.js';
+const US_SELECT_VALUE = 'us';
+
 // Whether the map is currently in sentiment mode or popularity mode.
 let isSentimentMode = true;
 
+// Whether the map is zoomed on world level or US states level.
+let isWorldLevel = true;
```

Comment:
What do you mean? Suggestion on a better name? I leave the possibility of zooming in to other countries and then this would need to change to a region code instead of a boolean, but don't think this is what you mean.